keyvi/src/cpp/dictionary/dictionary.h

```diff
@@ -70,7 +70,7 @@ final {
     }
 
     // temporary implementation
-    fsa::automata_t GetFsa() {
+    fsa::automata_t GetFsa() const {
```

Comment:
or we can have a `const` qualifier, but with return type `std::shared_ptr<const Automata>` which assures read-only behavior